db/view/view.cc

```diff
@@ -1609,10 +1609,13 @@ static future<> apply_to_remote_endpoints(service::storage_proxy& proxy, locator
         gms::inet_address target, inet_address_vector_topology_change&& pending_endpoints,
         frozen_mutation_and_schema&& mut, const dht::token& base_token, const dht::token& view_token,
         service::allow_hints allow_hints, tracing::trace_state_ptr tr_state) {
-
+    // The "delay_before_remote_view_update" injection point can be
+    // used to add a short delay (currently 0.5 seconds) before a base
+    // replica sends its update to the remove view replica.
+    co_await utils::get_local_injector().inject("delay_before_remote_view_update", 500ms);
```

Comment:
remove -> remote

Reply:
I'll fix this in my followup PR, thanks.